app/Concerns/IdentifiesIfACommentNeedsNotification.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Concerns;
+
+use App\NotifiedComment;
+use App\User;
+
+trait IdentifiesIfACommentNeedsNotification
+{
+    private function commentNeedsNotification($comment, User $user)
+    {
+        return ! (
+            $comment['updated_at'] < $user->created_at
+            || $comment['user']['id'] == $user->github_id
+            || NotifiedComment::where('github_id', $comment['id'])->count() > 0
```

Comment:
Is this line necessary anymore? theoretically no, right?

Reply:
Sorry, which line? (it shows all 3)

Reply:
oops, sorry--line 15. The GitHub lookup. It duplicates the eager load, right?

Reply:
(This is shared logic that is used in the eager load to check, so it is needed there)